integreat_cms/cms/forms/events/event_filter_form.py

```diff
@@ -94,9 +94,14 @@ def apply(self, events, region, language_slug):
             pass
         elif events_time_range.CUSTOM in cleaned_time_range:
             # Filter events for their start and end
-            events = events.filter_upcoming(
-                self.cleaned_data["date_from"] or date.min
-            ).filter(end_date__lte=self.cleaned_data["date_to"] or date.max)
+            tzinfo = zoneinfo.ZoneInfo(region.timezone)
```

Comment:
I guess we need to `import zoneinfo` in this file.

Reply:
Resolved